src/components/Card/Card.test.tsx

```diff
@@ -11,23 +11,23 @@ const itemWithImage = { title: 'This is a movie', duration: 120, cardImage: 'htt
 
 describe('<Card>', () => {
   it('renders card with video title', () => {
-    const { getByText } = renderWithRouter(<Card item={item} onClick={() => ''} />);
+    const { getByText } = renderWithRouter(<Card item={item} onClick={() => ''} url="" />);
```

Comment:
Let's put some dummy url in so we see it on the snapshots
```suggestion
    const { getByText } = renderWithRouter(<Card item={item} onClick={() => ''} url="https://test.dummy.jwplayer.com" />);
```